packages/ag-grid-styles/src/agGridStyles/styles.scss

```diff
@@ -1,6 +1,5 @@
-@import '~ag-grid-community/src/styles/ag-grid.scss';
-@import '~ag-grid-community/src/styles/webfont/agGridAlpineFont.scss';
```

Comment:
is the font no longer required?

Reply:
The package ``@ag-grid-commiunity/core`` imports the font when adding the sccs partial``ag-theme-alpine/sass/ag-theme-alpine-mixin``.